package.json

```diff
@@ -9,13 +9,10 @@
 		"test:js": "TS_NODE_FILES=true ava --serial # Due to Travis",
 		"build": "webpack --mode=production",
 		"watch": "webpack --mode=development --watch",
-		"watch:firefox": "web-ext run --source-dir=distribution",
```

Comment:
@fregante Was it a mistake that you deleted this line? I always use it for development.

Reply:
I removed it because of this https://github.com/sindresorhus/refined-github/pull/2683#issuecomment-572716508